histogram/histogram.go

```diff
@@ -26,7 +26,7 @@ type Histogram struct {
 
 func New(config Config) (*Histogram, error) {
 	if len(config.BucketLimits) == 0 {
-		return nil, microerror.Maskf(invalidConfigError, "%T.Buckets must not be empty", config)
+		return nil, microerror.Maskf(invalidConfigError, "%T.BucketLimits must not be empty", config)
```

Comment:
Ninja

Reply:
![giphy](https://user-images.githubusercontent.com/552769/43950843-ce215caa-9c91-11e8-8db1-22e933d7db9c.gif)

Reply:
How to hide that?

Reply:
`cmd` + `w`

Reply:
Or shuriken into the display.